RLTSquare/EmailTask/etc/di.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="RLTSquare\EmailTask\Logger\Handler">
```

Comment:
Overcomplicated code - the code around setting custom file name / logger is unnecessary overcomplicated. See the https://drive.google.com/file/d/1ed3KeOMXJoXriWg7CJeEh4rEGzFkU1vk/view?usp=share_link . There is no need to write any separate logger/handler files.